dali/python/nvidia/dali/plugin/paddle.py

```diff
@@ -70,13 +70,16 @@ def feed_ndarray(dali_tensor, ptr, cuda_stream = None):
                     Tensor from which to copy
     `ptr` : LoDTensor data pointer
             Destination of the copy
-    `cuda_stream` : Any value that can be casted to cudaStream_t
+    `cuda_stream` : cudaStream_t handle or any value that can be cast to cudaStream_t
                     CUDA stream to be used for the copy
                     (if not provided, an internal user stream will be selected)
     """
+
+    cuda_stream = types._raw_cuda_stream(cuda_stream)
+
     c_type_pointer = ctypes.c_void_p(ptr)
     if isinstance(dali_tensor, (TensorGPU, TensorListGPU)):
-        dali_tensor.copy_to_external(c_type_pointer, cuda_stream)
+        dali_tensor.copy_to_external(c_type_pointer, ctypes.c_void_p(cuda_stream))
```

Comment:
Now you will get nullptr instead of None. This breaks the logic in:
```
    .def("copy_to_external",
        [](Tensor<GPUBackend> &t, py::object p, py::object cuda_stream, bool non_blocking) {
          void *ptr = ctypes_void_ptr(p);
          cudaStream_t stream = cuda_stream.is_none()
                ? UserStream::Get()->GetStream(t)
                : static_cast<cudaStream_t>(ctypes_void_ptr(cuda_stream));
```